lms/envs/common.py

```diff
@@ -2754,6 +2754,7 @@ def _make_locale_paths(settings):  # pylint: disable=missing-function-docstring
     'ROOT': 'root',
     'TOS': 'tos',
     'HONOR': 'honor',  # If your site does not have an honor code, simply delete this line.
+    'TOS_AND_HONOR': 'edx-terms-service',
```

Comment:
This is to fix the view certificate page locally since it receives a NameError if this isn't defined.